payloads/cpg-soap-payload.xml

```diff
@@ -0,0 +1,161 @@
+<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope"
+               xmlns:ns35="http://www.justice.gov.uk/magistrates/external/ExternalDocumentRequest">
+    <soap:Header xmlns:wsa="http://www.w3.org/2005/08/addressing">
+        <wsa:Action>externalDocument</wsa:Action>
+        <wsa:From>
+            <wsa:Address>CP_NPS_ML</wsa:Address>
+        </wsa:From>
+        <wsa:MessageID>09233523-345b-4351-b623-5dsf35sgs5d6</wsa:MessageID>
+        <wsa:RelatesTo>RelatesToValue</wsa:RelatesTo>
+        <wsa:To>CP_NPS</wsa:To>
+    </soap:Header>
+    <soap:Body>
+        <ns35:ExternalDocumentRequest>
+            <documents>
+                <document>
+                    <info>
+                        <general>
+                            <doc_type>EDT_Adult_Daily_Courtlist</doc_type>
+                            <outputtype>R</outputtype>
+                            <version>1_0</version>
+                            <docref>575</docref>
+                            <system>L</system>
+                        </general>
+                        <source_file_name>5_25052021_2992_B14LO00_ADULT_COURT_LIST_DAILY</source_file_name>
+                        <printers/>
+                        <parameters>
+                            <pi_mcc_id>7</pi_mcc_id>
+                            <pi_psa_code>2360</pi_psa_code>
+                            <pi_job_type>ADULT_COURT_LIST_DAILY</pi_job_type>
+                            <pi_run_date>26/11/2020</pi_run_date>
+                            <pi_edt_user>2368</pi_edt_user>
+                        </parameters>
+                        <start_time>09:05:04</start_time>
+                    </info>
+                    <data>
+                        <job>
+                            <printdate>14/11/2016</printdate>
+                            <username>gl.userone</username>
+                            <late>N</late>
+                            <adbox>N</adbox>
+                            <means>N</means>
+                            <sessions>
+                                <session>
+                                    <s_id>556805</s_id>
+                                    <doh>27/09/2021</doh>
```

Comment:
This needs to be changed dynamically in order to generate a list with the current date.

If you change the content to TODAY_DATE this can be replaced within the route:

```
const todayDate = new Intl.DateTimeFormat('en-GB').format(new Date())
const soapEnvelope = fs.readFileSync(path.join(process.cwd(), './payloads/cpg-soap-payload.xml'), 'utf-8').replace('TODAY_DATE', todayDate)
```

Reply:
Having chatted with Sarah we decided to break that out into a [separate ticket](https://dsdmoj.atlassian.net/browse/PIC-1553) so that Sarah can focus on getting what's there tested and tidied up. That snippet is worth keeping though so will link to this comment in the ticket!

Reply:
Ah ok, sorry. Was just checking it against the story. Carry on. 😃